### PR TITLE
Add RDS for rule 15-8

### PR DESCRIPTION
**Pull request by mcneillj**

### CODE REVIEW

File: docs/section15/Rule15-8.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+**Rule Description:** Transformer efficiency reported in Proposed RMR equals User RMR  
+**Rule Assertion:** User RMR = Proposed RMR  
+**Appendix G Section:** Transformers  
+**Appendix G Section Reference:**  Table G3.1 15. Distribution Transformers  
+**Data Lookup:** Table 8.4.4  
+**Evaluation Context:**  Each Data Element   
+**Applicability Checks:**
+1. Transformer is DRY_TYPE
+2. U_RMR transformer capacity is within range of Table 8.4.4  
+
```
Comment (mkarpman) [inline docs/section15/Rule15-8.md:12]: Another applicability check is that tranformer efficiency exceeds minimum in Table 8.4.4.

File: docs/section15/Rule15-8.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+**Manual Checks:** None  
+
+## Rule Logic:
+- For each transformer `_user_transformer` in `U_RMR.transformers`:
+    - Get maximum regulated capacity: `if _user_transformer.phase == SINGLE_PHASE: _max_capacity_limit = 333.0 else if _user_transformer.phase == THREE_PHASE: _max_capacity_limit = 1000.0`  
+    - **Applicability Check 1:** `_user_transformer.type == DRY_TYPE`  
+    - **Applicability Check 2:** `_user_transformer.capacity >= 15.0 and _user_transformer.capacity <= _max_capacity_limit:`
+    - Get proposed transformer matching name of user transformer: `_proposed_transformer = match_data_element(P_RMR, transformers, _user_transformer.name)`
+    - **Rule Assertion:** `_user_transformer.efficiency == _proposed_transformer.efficiency`
```
[the remark this answers is not in the document]
Comment (mkarpman) [reply docs/section15/Rule15-8.md:21]: Agreed

[the remark this answers is not in the document]
Comment (mkarpman) [reply docs/section15/Rule15-8.md:21]: Also need to change to PASS/FAIL outcome framework that we used for envelope and lighting

**Comment by weilixu** [comment]
No longer relevant - close it now.

**Pull request closed without merging**